client/src/components/ConfigTemplates/InstanceDropdown.vue

```diff
@@ -42,7 +42,7 @@ const emit = defineEmits<{
                 :href="routeUpgrade"
                 @keypress="router.push(routeUpgrade)"
                 @click.prevent="router.push(routeUpgrade)">
-                <FontAwesomeIcon icon="arrowUp" />
+                <FontAwesomeIcon icon="arrow-up" />
```

Comment:
Nice trick/recommendation:
Instead of translating the icon's ID to a string, you can directly reference the imported icon using the following...

```suggestion
                <FontAwesomeIcon :icon="faArrowUp" />
```
I find this approach easier to maintain and "grep" because it matches the same name as it was imported. Also, your IDE will warn you if you spell it wrong.

Reply:
Neat - thank you so much!